### ISSUE TITLE
OdooModuleServiceImpl.getModule IndexNotReadyException

### ISSUE THREAD

**Issue by DieterReinert** [wt-io-it/odoo-pycharm-plugin#104]
Plugin Version: 0.6.7
IntelliJ Version: PyCharm 2022.3
IntelliJ Build: PY-223.7571.203

<details><summary>com.intellij.openapi.project.IndexNotReadyException: Please change caller according to com.intellij.openapi.project.IndexNotReadyException documentation
</summary>

```
com.intellij.openapi.project.IndexNotReadyException: Please change caller according to com.intellij.openapi.project.IndexNotReadyException documentation
	->at com.intellij...create(IndexNotReadyException.java:66)
	...
	->at com.intellij...getValues(FileBasedIndexEx.java:139)
	at at.wtioit.intellij.plugins.odoo.index.OdooIndex.getValues(OdooIndex.java:49)
	at at.wtioit.intellij.plugins.odoo.modules.impl.OdooModuleServiceImpl.lambda$getModule$1(OdooModuleServiceImpl.java:63)
	->at com.intellij...runReadAction(ApplicationImpl.java:860)
	at at.wtioit.intellij.plugins.odoo.modules.impl.OdooModuleServiceImpl.getModule(OdooModuleServiceImpl.java:61)
	at at.wtioit.intellij.plugins.odoo.pycharm.PyCharmOdooAddonsImportResolver.resolveImportReference(PyCharmOdooAddonsImportResolver.java:35)
	at at.wtioit.intellij.plugins.odoo.pycharm.PyCharmOdooAddonsImportResolver.resolveImportReference(PyCharmOdooAddonsImportResolver.java:41)
	->at com.jetbrains...invoke(PyResolveImportUtil.kt:199)
	...
	->at com.intellij...beforeDocumentChange(SuggestedRefactoringChangeListener.kt:210)
	at jdk.internal.reflect.GeneratedMethodAccessor204.invoke(Unknown Source)
	->at java...invoke(DelegatingMethodAccessorImpl.java:43)
	...
	->at com.intellij...lambda$createMulticaster$1(EventDispatcher.java:85)
	at jdk.proxy2/jdk.proxy2.$Proxy102.beforeDocumentChange(Unknown Source)
	->at com.intellij...lambda$beforeChangedUpdate$1(DocumentImpl.java:881)
	...
	->at com.intellij...invoke(MessageBusImpl.kt:421)
	at jdk.proxy2/jdk.proxy2.$Proxy186.after(Unknown Source)
	->at com.intellij...fireAfterEvents(PersistentFSImpl.java:1208)
	...
	->at java...run(EventDispatchThread.java:92)
```
</details>

<details><summary>induced.txt</summary>

```
Following exceptions happened soon after this one, most probably they are induced.


2022-12-16T08:33:53.623
com.intellij.openapi.project.IndexNotReadyException: Please change caller according to com.intellij.openapi.project.IndexNotReadyException documentation
	->at com.intellij...create(IndexNotReadyException.java:66)
	...
	->at com.intellij...getValues(FileBasedIndexEx.java:139)
	at at.wtioit.intellij.plugins.odoo.index.OdooIndex.getValues(OdooIndex.java:49)
	at at.wtioit.intellij.plugins.odoo.modules.impl.OdooModuleServiceImpl.lambda$getModule$1(OdooModuleServiceImpl.java:63)
	->at com.intellij...runReadAction(ApplicationImpl.java:860)
	at at.wtioit.intellij.plugins.odoo.modules.impl.OdooModuleServiceImpl.getModule(OdooModuleServiceImpl.java:61)
	at at.wtioit.intellij.plugins.odoo.pycharm.PyCharmOdooAddonsImportResolver.resolveImportReference(PyCharmOdooAddonsImportResolver.java:35)
	at at.wtioit.intellij.plugins.odoo.pycharm.PyCharmOdooAddonsImportResolver.resolveImportReference(PyCharmOdooAddonsImportResolver.java:41)
	->at com.jetbrains...invoke(PyResolveImportUtil.kt:199)
	...
	->at com.intellij...beforeDocumentChange(SuggestedRefactoringChangeListener.kt:210)
	at jdk.internal.reflect.GeneratedMethodAccessor204.invoke(Unknown Source)
	->at java...invoke(DelegatingMethodAccessorImpl.java:43)
	...
	->at com.intellij...lambda$createMulticaster$1(EventDispatcher.java:85)
	at jdk.proxy2/jdk.proxy2.$Proxy102.beforeDocumentChange(Unknown Source)
	->at com.intellij...lambda$beforeChangedUpdate$1(DocumentImpl.java:881)
	...
	->at java...run(EventDispatchThread.java:92)

```
</details>

**Comment by bigbear3001** [comment]
this is fixed with #67 in 0.6.10 (pending jetbrains approval for the marketplace) if you are eager to test it you can use the jar file from https://github.com/wt-io-it/odoo-pycharm-plugin/tree/v0.6.10

**Comment by ap-wtioit** [comment]
0.6.10 is available in the marketplace now
